CodeNeninja/Adventure-Time
# Mode3

# Created the third mode

## What's new?
- RandomEvent = Get a random number from 1 to 3 for the other event in the functions.
- AdventureMode = The main function for random events. It picks a random number (RandomEvent()). And it runs a random event.
- Fight event = Just a fightevent with a tiger or flee.
- TreasuerRisk = You can get a survival pack or eaten by sharks.
- ShelterEvent = You can fully rest or you are bitten by a snake.

## How to test:
- Switch to my branch.
- Run the project.
- See if it works.
- Please check if my code is clean.
